src/Wellcome.Dds/Wellcome.Dds.Dashboard/Controllers/DashGlobalsActionFilter.cs

```diff
@@ -30,7 +31,20 @@ public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionE
             viewBag.HeartbeatClass = warningState ? "btn-danger" : "";
             var getQueueLevel = dashboardRepository.GetDlcsQueueLevel();
```

Comment:
Not a comment on this PR, but this is the sort of thing that in the dashboard could be filled by a client-side call to a controller... pop in later rather than delay page rendering. Same is true of some other bits of the page. Just an ideal world thing, not worth it for now.